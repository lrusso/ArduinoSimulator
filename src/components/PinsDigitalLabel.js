import React from "react"
import { t } from "../utils/service"

const PinsDigitalLabel = () => {
  return <div style={styles.label}>{t("DIGITAL_PINS")}</div>
}

const styles = {
  label: {
    float: "left",
    fontFamily: "Arial",
    fontSize: "13px",
    lineHeight: 2,
    fontWeight: "bold",
    marginLeft: "7px",
    marginRight: "1px",
    cursor: "default",
  },
}

export default PinsDigitalLabel
