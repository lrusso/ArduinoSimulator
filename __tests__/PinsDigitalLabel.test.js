import React from "react"
import renderer from "react-test-renderer"
import PinsDigitalLabel from "../src/components/PinsDigitalLabel"

describe("PinsDigitalLabel", () => {
  test("Rendering 1", () => {
    const tree = renderer.create(<PinsDigitalLabel />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
