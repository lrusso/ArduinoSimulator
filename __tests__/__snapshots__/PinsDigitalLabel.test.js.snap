// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PinsDigitalLabel Rendering 1 1`] = `
<div
  style={
    {
      "cursor": "default",
      "float": "left",
      "fontFamily": "Arial",
      "fontSize": "13px",
      "fontWeight": "bold",
      "lineHeight": 2,
      "marginLeft": "7px",
      "marginRight": "1px",
    }
  }
>
  DIGITAL PINS
</div>
`;
